frontend/src/app/progress-bar/progress-bar.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'volontulo-progress-bar',
+  templateUrl: './progress-bar.component.html',
```

Comment:
You have removed stylesUrls but you also need to remove empty progress-bar.component.scss file.